controllers/schools.js

```diff
@@ -7,9 +7,32 @@ const express = require('express');
 const router = express.Router();
 const authHelper = require('../helpers/authentication');
 const api = require('../api');
-const moment = require('moment');
+const moment = require('moment-timezone');
+
 moment.locale('de');
 
+function timeConvert(num) {
+    let hours = (num / 60);
+    let rhours = Math.floor(hours);
+    let minutes = (hours - rhours) * 60;
+    rhours = rhours > 0 ? `+${rhours}` : rhours;
+    minutes = minutes === 0 ? '00' : minutes;
+    return `(UTC ${rhours}:${minutes})`;
+}
+let countryCodes = moment.tz.countries();
+countryCodes = countryCodes.map((item) => {
+    return moment.tz.zonesForCountry(item, true);
+});
+countryCodes = [].concat.apply([], countryCodes);
+countryCodes.sort((a, b) => parseFloat(a.offset) - parseFloat(b.offset));
```

Comment:
Please improve sorting for partial hours.

Reply:
done ;)